ASCII_SUBSEQUENCES/Ascii_subsequences.java

```diff
@@ -0,0 +1,42 @@
+import java.util.Scanner;
+
+public class Ascii_Subsequences {
+	
+    public static void main(String args[]) {
+	   
+        /* In this problem we have to find all the possible Ascii subsequences
+         * of the given string */
+    
+        Scanner obj = new Scanner(System.in);
+	String str = obj.next();
+	    
+	int ans = find(str, ""); 
+	
+	System.out.println();
+	System.out.println(ans); 
+	
+    }
+
+    public static int find(String str, String ans) {
+	    
+        if (str.length() == 0) {
+            
+            System.out.print(ans + " ");
+	        return 1;  		
```

Comment:
4 space indentation

Reply:
![image](https://user-images.githubusercontent.com/44175278/80214291-343edb00-8658-11ea-8e24-8e095bdb6fce.png)

Reply:
What is the problem in this @somya-kapoor

Reply:
![Screenshot (122)](https://user-images.githubusercontent.com/42409030/80395112-62c6eb00-88d0-11ea-99cf-f2d35ec9f11d.png)

Reply:
it looks like 2 space indentation